squidb-tests/src/com/yahoo/squidb/test/TestDatabase.java

```diff
@@ -83,6 +83,7 @@ protected void onOpen(ISQLiteDatabase db) {
     protected void onConfigure(ISQLiteDatabase db) {
         /** @see AttachDetachTest#testAttacherInTransactionOnAnotherThread() */
         db.enableWriteAheadLogging();
+        setFastInsertEnabled(true);
```

Comment:
We could do it similarly to how we do the SQLite binding thing, with a custom test runner. I can look into making that happen.